web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -894,31 +894,31 @@ input::-webkit-textfield-decoration-container {
 // reduces the space under the breadcrumb on the downlevel page with a leftnav to look even
 @media (max-width: 30rem) {
   .breadcrumb-spacing {
-    height: 2.5rem;
+    min-height: 2.5rem;
   }
 }
 
 @media (min-width: 30.0625em) and (max-width: 40em) {
   .breadcrumb-spacing {
-    height: 4rem;
+    min-height: 4rem;
   }
 }
 
 @media (min-width: 40.0625em) and (max-width: 63.93em) {
   .breadcrumb-spacing {
-    height: 1.7rem;
+    min-height: 1.7rem;
   }
 }
 
 @media screen and (min-width: 64em) {
   .breadcrumb-spacing {
-    height: 3.7rem;
+    min-height: 3.7rem;
   }
 }
 
 // reduces space below the breadcrumb on benefits pages
 #block-usagov-breadcrumbs {
-  height: 4rem;
+  min-height: 4rem;
 }
 
 #block-usagov-breadcrumbs {
```

Comment:
If I set my browser viewport size to 480px, and visit this page (http://localhost/dv-lottery-eligibility), the breadcrumbs still overlap. Changing the "height: 5rem" to a min-height would help in that scenario but I believe this style is intended to apply for the mobile version of the breadcrumbs which ends right before 480px, so it would be better to use the USWDS breakpoint utility with the mobile-lg token to replace lines 920-928 like so:
```
#block-usagov-breadcrumbs {
  min-height: 5rem;
  
  @include at-media("mobile-lg") {
    min-height: 4rem;
  }
}
```
We could probably benefit from using this breakpoint utility in many of our other responsive styles as well, but that would be for another ticket.

Reply:
Oops - I missed this somehow. This is now resolved on this branch.